homeassistant/components/linknlink/entity.py

```diff
@@ -0,0 +1,66 @@
+"""linknlink entities."""
+
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+
+class LinknLinkEntity(Entity):
```

Comment:
Why isn't this a coordinator Entity?

Reply:
I'm sorry for the trouble I caused. Please check: https://github.com/home-assistant/core/pull/107054